iosHyperskillApp/iosHyperskillApp/Sources/Modules/ProblemsLimit/Views/ProblemsLimitContent.swift

```diff
@@ -0,0 +1,52 @@
+import shared
+import SwiftUI
+
+extension ProblemsLimitContent {
+    struct Appearance {
+        let skeletonHeight: CGFloat = 40
+    }
+}
+
+struct ProblemsLimitContent: View {
+    private(set) var appearance = Appearance()
+
+    let stateKs: ProblemsLimitFeatureViewStateKs
+
+    var body: some View {
+        switch stateKs {
+        case .idle, .error:
+            EmptyView()
+        case .loading:
+            ProblemsLimitSkeletonView()
+        case .content(let content):
+            switch ProblemsLimitFeatureViewStateContentKs(content) {
+            case .empty:
+                EmptyView()
+            case .widget(let data):
+                ProblemsLimitWidgetView(
+                    stepsLimitLeft: Int(data.stepsLimitLeft),
+                    stepsLimitTotal: Int(data.stepsLimitTotal),
+                    stepsLimitLabel: data.stepsLimitLabel,
+                    updateInLabel: data.updateInLabel
+                )
+            }
+        }
+    }
```

Comment:
Why render differently compare with the current `develop` state?
<img width="559" alt="Screenshot 2023-05-09 at 2 36 22 PM" src="https://github.com/hyperskill/mobile-app/assets/8586063/114757a0-8c0a-4c3b-8387-b1dfcb03589d">